x-pack/plugins/apm/public/components/shared/is_route_with_time_range.ts

```diff
@@ -31,3 +31,27 @@ export function isRouteWithTimeRange({
 
   return matchesRoute;
 }
+
+export function isRouteWithComparison({
+  apmRouter,
+  location,
+}: {
+  apmRouter: ApmRouter;
+  location: Location;
+}) {
+  const matchingRoutes = apmRouter.getRoutesToMatch(location.pathname);
+  const matchesRoute = matchingRoutes.some((route) => {
+    return (
+      route.path === '/services' ||
+      route.path === '/service-map' ||
+      route.path === '/backends' ||
+      route.path === '/backends/inventory' ||
+      route.path === '/services/{serviceName}' ||
+      route.path === '/service-groups' ||
+      location.pathname === '/' ||
+      location.pathname === ''
+    );
+  });
```

Comment:
Would be great if we could determine whether a route had comparisons, without duplicating the path names. 
eg.

```suggestion
  const matchingRoutes = apmRouter
    .getRoutesToMatch(locations.pathname)
    .filter((route) => route.params.query.comparisonEnabled);
    
  return matchingRoutes.length > 0;    
```

Reply:
@dgieselaar is there another method to get the params based on path / location?

Reply:
I think this is obscured by `io-ts` unfortunately.